eml/api/api.js

```diff
@@ -42,15 +40,21 @@ export const getCourseWithAuth = async (courseId) => {
 
 export const getCourses = async () => {
   // TODO: add bearer token to request header and omit /public
-  const res = await axios.get(url + '/api/public/courses');
+  const res = await axios.get(url + '/api/courses');
+  return res.data;
```

Comment:
Microsoft convention requires endpoint to be singular. (not plural) 
So /api/course

Reply:
+1